### DISCUSSION TITLE
EDUCATION PACK HELP

### DISCUSSION THREAD

**Discussion by FBIopeningUPP** [community/community#135517] [GitHub Education]
### Hi everyone,

I applied for GitHub education pack and got it approved but never received any mail about it and didn't receive my stuff included in that

**Comment by davevad93** [comment]
Hi @erikaSKOOL ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @erikaSKOOL was approved and your benefits are now available.`

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; waiting times are approximate and may vary depending on different factors, such as the volume of requests. 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135145

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
